### ISSUE TITLE
Usage of @splat

### ISSUE THREAD

**Issue by mjoork** [Jack-Ji/jok#1]
Right now, if I use any of the functions that use `@splat`, it causes a compile error:
```
/home/mpokhil/Source/jok/src/j2d/Vector.zig:97:39: error: expected type 'u32', found 'type'
    const result = self.data / @splat(f32, l);
                                      ^~~
referenced by:
    move: /home/.../src/main.zig:63:30
    update: /home/.../src/main.zig:146:13
    remaining reference traces hidden; use '-freference-trace' to see all reference traces

/home/mpokhil/Source/jok/src/j2d/Vector.zig:23:27: error: expected type 'u32', found 'type'
    const result = @splat(f32, val);
```

Just changing all instance of `@splat(f32, ...)` in `jok.j2d.Vector` to `@splat(2, ...)` works as expected.

Should I create a PR for this?

**Comment by Jack-Ji** [comment]
Thanks for big report, I’ll give it a check as soon as possible.

**Comment by Jack-Ji** [comment]
Fixed 5aedc0e.
